eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -105,6 +105,7 @@ stages:
         container: ${{ parameters.centOSStream9Container }}
         bootstrapPrep: false          # 🚫
         enablePoison: false           # 🚫
+        enableSha1Signing: true       # ✅
```

Comment:
The convention used has been to specify all parameters in all legs.  It was done to make it easier to see the differences between the various legs and to support configuration changes (including duplicating a leg and tweaking it).  

This option feels a bit different in that it should only be used on CentOSStream legs but I see value in continuing the current pattern.  I don't feel strongly about this though.

Reply:
If other distros specify `enableSha1Signing: false`, a reader could infer that Sha1 signing is disabled on those platforms; it's not.

I generally dislike negative variable names but in this case maybe something like `overrideDistroDisablingSha1` might be more accurate if we are going to set it to `false` for other entries in our build matrix?

Reply:
`forceSha1Signing` is an option too?